stable/jenkins/templates/_helpers.tpl

```diff
@@ -142,6 +142,11 @@ jenkins:
         yaml: |-
           {{ tpl .Values.agent.yamlTemplate . | nindent 10 | trim }}
         yamlMergeStrategy: "override"
+      {{- if .Values.agent.podTemplates }}
+        {{- range $key, $val := .Values.agent.podTemplates }}
+          {{ tpl $val $| nindent 6 }}
```

Comment:
how about `{{-` to avoid the blank line and `$ |` with a space

Reply:
Tomorrow I'll change/test this! thank you so much for your recommendations!

Reply:
Thank you also for this correction! done already.